### PR TITLE
blog/data/authors/michal.zygowski.json: update BIO

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

**Comment by artur-rs** [comment]
@miczyg1 LGTM

File: blog/content/authors/michal-zygowski/_index.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
+facebook: "https://www.facebook.com/miczyg1395"
+twitter: "https://twitter.com/_miczyg_"
+linkedin: "https://www.linkedin.com/in/miczyg"
 github: "https://github.com/miczyg1"
 ---
 Firmware Engineer with networking background. Feels comfortable with low-level
-development using C/C++, RTOS (especially ARM mbed OS) and writing drivers for
-small microcontrollers. Experienced with STM32 MCUs and coreboot project.
+development using C/C++ and assembly. Interested in advanced hardware features,
+security and coreboot. Core coreboot developer, Braswell SoC, PC Engines,
```
Comment (pietrushnic) [inline blog/content/authors/michal-zygowski/_index.md:12]:
```suggestion
security and coreboot. Core developer of coreboot. Maintainer of Braswell SoC, PC Engines,
```

File: blog/content/authors/michal-zygowski/_index.md [hunk 2]
```diff
@@ -2,11 +2,12 @@
...
+Protectli and Libretrend platforms maintainer.
```
Comment (pietrushnic) [inline blog/content/authors/michal-zygowski/_index.md:13]:
```suggestion
Protectli and Libretrend platforms.
```

File: blog/content/authors/michal-zygowski/_index.md (hunk 1 above)
Comment (miczyg1) [reply blog/content/authors/michal-zygowski/_index.md]: Done

File: blog/content/authors/michal-zygowski/_index.md (hunk 2 above)
Comment (miczyg1) [reply blog/content/authors/michal-zygowski/_index.md]: Done

**Pull request merged**